src/Microsoft.Azure.WebJobs.Host/ConfigurationUtility.cs

```diff
@@ -34,6 +35,12 @@ public static void SetConfigurationFactory(Func<IConfiguration> configurationRoo
             Reset();
         }
 
+        public static bool IsDisabledMethod(string methodName)
+        {
+            var disabledSetting = GetSetting(string.Format(CultureInfo.InvariantCulture, "{0}.Disabled", methodName));
+            return string.Compare(disabledSetting, "true", StringComparison.OrdinalIgnoreCase) == 0;
```

Comment:
Consider also allowing 1/0 in addition to true/false. We already have this logic in HostListenerFactory.IsDisabledBySetting. Recommend just factoring that logic out into a helper on ConfigurationUtility that is shared